lagom14-java-conductr-bundle-lib/build.sbt

```diff
@@ -0,0 +1,30 @@
+import Tests._
+
+name := "lagom14-java-conductr-bundle-lib"
+
+libraryDependencies ++= List(
+  Library.lagom14ClientJavadsl,
+  Library.akka25Testkit % "test",
+  Library.play26Test    % "test",
+  Library.scalaTest     % "test"
+)
+
+crossScalaVersions := crossScalaVersions.value.filterNot(_.startsWith("2.12"))
```

Comment:
You should get rid of this line. Its presence means that they'd be no Scala 2.12 cross build - you want that for 1.4.